in_text/text_02_drivers.yml

```
@@ -117,7 +117,7 @@ entities:
     -
       attr-label: Rain
       attr-def: >-
-        Daily rate of rain reaching the reservoir surface in a day as described in Winslow and others 2017; rate as determined by Hipsey and others 2019.
+        Daily rate of rain reaching the reservoir surface in a day as described in Winslow and others 2017; rate as determined by Hipsey and others 2019. 
```

Need to add reference for Hipsey and others 2019
```suggestion
        Daily rate of rain reaching the reservoir surface in a day as described in Winslow and others 2017; rate as determined by Hipsey and others 2019.
```

---

added the citation here https://github.com/USGS-R/forecast-data-release-v2/pull/13/commits/2c3dd1626037b8e07abd9416d5b7fcacb82500cc to `text_02_drivers.yml`. @limnoliver do let me know if this citation should be placed elsewhere in the release.